common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -139,8 +138,8 @@ def create_pull_request(repo, pr_name, source_branch, target_branch,
         pr_path = "#{repo}/pullrequests"
         post(base_url + pr_path, content.to_json)
       end
-      # rubocop:enable Metrics/ParameterLists
 
+      # rubocop:enable Metrics/ParameterLists
```

Comment:
This should go back under the `end`, it closes out the disable pragma from line 119 and wraps that method